httpe/httpe.go

```diff
@@ -59,6 +59,25 @@ func NewHandlerFunc(h HandlerFuncE, ew ErrWriterFunc) http.HandlerFunc { //nolin
 	return NewHandler(h, ew).(http.HandlerFunc)
 }
 
+// NewDefaultHandler creates a new http.HandlerFunc which calls
+// HandlerE.ServeHTTP and if it returns an error calls WriteHTTPeErr
+// to create the appropriate response.
+func NewDefaultHandler(h HandlerE) http.Handler {
+	f := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
The body of this function could be `return NewHandler(h, WriteHTTPeErr)`

Reply:
Yes! (`return NewHandler(h, ErrWriterFunc(WriteHTTPeErr))`)

Reply:
done.